Add section about css custom properties (#1640)
wix/stylable: packages/stylable.io/docs/guides/migration-v3.md

> `st`. It is recommended to use `st` in order to avoid name clashing
> with other variables (for example, some other inline style).

### CSS custom properties 

> Stylable now also localize css custom properties, it means that any usage of custom proprties (--prop) should be now incoparated in the component code.

Reviewer: ```suggestion
Stylable now localizes CSS Custom Properties. This means that any usage of Custom Proprties (e.g. `--prop`) should now be incorparated in the component code.
```

> See the docs about [CSS custom properties](https://stylable.io/docs/references/css-vars) for usage.

Reviewer: ```suggestion
See the docs about [CSS custom properties](https://stylable.io/docs/references/css-vars) for usage examples.
```
